fiftyone/utils/cityscapes.py

```
@@ -202,10 +214,13 @@ def _parse_split(split):
     if split == "validation":
         return "val"
 
+    if split == "train_extra":
+        return "train_extra"
+
     if split not in ("test", "train"):
         raise ValueError(
             "Invalid split '%s''; supported values are %s"
-            % (split, ("train", "test", "validation"))
+            % (split, ("train", "train_extra", "test", "validation"))
```

Ruff (0.15.12)

[warning] Use format specifiers instead of percent format

Replace with format specifiers

(UP031)